src/browser/browser.zig

```diff
@@ -581,10 +582,18 @@ pub const Page = struct {
     // It resolves src using the page's uri.
     // If a base path is given, src is resolved according to the base first.
     // the caller owns the returned string
+    // TODO: We are leaking the memory associated with the response.
```

Comment:
We leverage arena allocators a lot, hence the name of the variable is `arena`. It's cleared whenever a page is created in `createPage`. So it isn't leaking, it's just [probably] living for longer than it absolutely has to.

Reply:
Makes sense